src/test/java/com/wikia/webdriver/testcases/discussions/DeleteAllPostsByUserTests.java

```diff
@@ -0,0 +1,239 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.common.remote.Discussions;
+import com.wikia.webdriver.common.remote.operations.DiscussionsOperations;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.DeleteAllButton;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.DeleteDialog;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.Post;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
+import com.wikia.webdriver.elements.mercury.pages.discussions.UserPostsPage;
+import org.testng.annotations.*;
+
+
+import static com.wikia.webdriver.elements.mercury.components.discussions.common.DiscussionsConstants.DESKTOP_RESOLUTION;
+import static com.wikia.webdriver.common.core.Assertion.assertTrue;
+import static com.wikia.webdriver.common.core.Assertion.assertFalse;
+
+
+/**
+ * Tests for deleting all posts by some user
+ */
+@Execute(onWikia = MercuryWikis.DISCUSSIONS_AUTO)
+public class DeleteAllPostsByUserTests extends NewTestTemplate {
+
+  private final User USER_WITH_POSTS = User.USER_6;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "USER_WITH_POSTS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)